src/condor_tests/test_custom_machine_resource_instances.py

```diff
@@ -11,7 +11,9 @@
 # want to test the pieces individually, you'll have to change the code.
 #
 
+import pytest 
```

Comment:
This doesn't seem to be necessary.  (Which means it's superfluous in `test_custom_machine_resource.py`, but that's for later.  Not a blocker.